src/store/notification/index.ts

```diff
@@ -0,0 +1,87 @@
+import Vue from 'vue'
+import {ActionTree, Module} from 'vuex'
+import {IRootState} from '@/store'
+import {ToastOptions} from 'vue-toastification/dist/types/src/types'
+import {POSITION} from 'vue-toastification/src/ts/constants'
+import {TYPE} from 'vue-toastification/dist/types/src/ts/constants'
+
+/**
+ * For more details about available options, see https://vue-toastification.maronato.dev/
+ */
+const GENERAL_OPTIONS: ToastOptions = {
+  position: POSITION.TOP_RIGHT,
+  draggable: true,
+  draggablePercent: 0.6,
+  closeOnClick: false,
+  timeout: false,
+}
+
+/**
+ * Note, we can use a custom component for `content`, not only texts
+ */
+export type IToastContent = string | object
+
+export interface IToast {
+  content: IToastContent,
+  options: ToastOptions,
+}
```

Comment:
Would it be okay to move IToast _above_ IToastContent, maybe even to the very top of the file?
That way the reader would see the higher-level type first